guides/doc-Planning_Guide/topics/Capsule_Server_Overview.adoc

```diff
@@ -20,7 +20,7 @@ Infrastructure and host management services:
 
 * *DHCP* – {SmartProxy} can act as a DHCP server or it can integrate with an existing solution, including ISC DHCP servers, Active Directory, and Libvirt instances.
 
-* *DNS* – {SmartProxy} can act as a DNS server or it can integrate with an existing solution, including ISC DNS, Active Directory, or BIND.
+* *DNS* – {SmartProxy} can act as a DNS server or it can integrate with an existing solution, including ISC BIND or Active Directory.
```

Comment:
```suggestion
* *DNS* – {SmartProxy} can act as a DNS server or it can integrate with an existing solution, including ISC BIND and Active Directory.
```
You are totally correct about using `and` here, it works better.

Reply:
Technically the Smart Proxy provider we use is using nsupdate so any [RFC2136](https://tools.ietf.org/html/rfc2136) compatible nameserver can be used. ISC bind is the one where we can also manage the server.

IMHO `{SmartProxy} can act as a DNS server is incorrect. The Smart Proxy can only integrate with DNS servers. The installer can deploy ISC bind on the same server. The same goes for DHCP and TFTP.